src/screens/CreatePost/index.js

```diff
@@ -844,13 +850,15 @@ const CreatePost = () => {
             topics={listTopic}
             chatTopics={listTopicChat}
             onClose={() => sheetTopicRef.current.close()}
+            isAnonimity={typeUser}
             // saveOnClose={(v, chatTopic) => onSaveTopic(v, chatTopic)}
           />
           <SheetExpiredPost
             refExpired={sheetExpiredRef}
             data={postExpired}
             select={expiredSelect}
             onSelect={onSetExpiredSelect}
+            isAnonimity={typeUser}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, ensure that the `isAnonimity` prop is being used in the components.
```diff
-            isAnonimity={typeUser}
+            isAnonymous={typeUser}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->